lib/elastomer/client/percolator.rb

```diff
@@ -0,0 +1,72 @@
+module Elastomer
+  class Client
+
+    class Percolator
+
+      # Create a new Percolator for managing a query.
+      #
+      # client - Elastomer::Client used for HTTP requests to the server
+      # index  - The index name
+      # id     - The _id for the query
+      def initialize(client, index, id)
+        @client = client
+        @index = index
+        @id = id
```

Comment:
Use `client.assert_param_presence` to verify that these are valid values. See https://github.com/github/elastomer-client/blob/a19ffa5ab3207ecf162f1a37e70399bad1e31eb7/lib/elastomer/client/warmer.rb#L24-L25